apps/omg_child_chain/test/omg_child_chain/integration/happy_path_test.exs

```diff
@@ -209,6 +209,41 @@ defmodule OMG.ChildChain.Integration.HappyPathTest do
     assert {:ok, _} = submit_transaction(second_output_spend)
   end
 
+  @tag fixtures: [:alice, :in_beam_child_chain, :alice_deposits]
+  test "check in-flight exit input piggybacking doesn't crash",
```

Comment:
What does this test actually test? Just not crashing is a bit obscure definition of a test. Under which conditions?

Reply:
ah, absolutely right, there's a better way to word it, fixing...